widget/embedded/src/utils/time.ts

```diff
@@ -47,31 +55,42 @@ export function timeSince(millisecond: number): string {
   );
 
   for (const interval of sortedIntervals) {
-    const { turningPoint, label, pluralLabel } = interval;
+    const { turningPoint, label, pluralLabel, isExact } = interval;
     const intervalCount = Math.floor(seconds / turningPoint);
     if (intervalCount > 1) {
-      return `${intervalCount} ${pluralLabel}`;
-    }
-    if (intervalCount === 1) {
-      return `${intervalCount} ${label}`;
+      return {
+        time: isExact
+          ? `${new Date(millisecond).toLocaleDateString()} ${new Date(
+              millisecond
+            ).toLocaleTimeString()}`
+          : `${intervalCount} ${pluralLabel}`,
+        isExact,
+      };
+    } else if (intervalCount === 1) {
+      return {
+        time: isExact
+          ? `${new Date(millisecond).toLocaleDateString()} ${new Date(
+              millisecond
+            ).toLocaleTimeString()}`
+          : `${intervalCount} ${label}`,
+        isExact,
+      };
     }
   }
 
   if (seconds > 1) {
-    return `${seconds} ${i18n.t('seconds')}`;
+    return { time: `${seconds} ${i18n.t('seconds')}` };
   }
 
-  return `${seconds} ${i18n.t('second')}`;
+  return { time: `${seconds} ${i18n.t('second ago')}` };
```

Comment:
second ago => seconds ago

Reply:
fixed. it shouldn't have "ago", because it is provided in getSwapDate function.